src/main/perl/Process.pm

```diff
@@ -103,6 +103,16 @@ regardless of any value for C<NoAction>.
 By default, commands modify the state and thus C<keeps_state> is
 false.
 
+=item C<sensitive>
+
+A boolean specifying whether the command (and args) contain sensitive information
+(like passwords). If C<sensitive> is true, the commandline will not be reported
+(by default the commanlione is reported in with verbose).
+
+This does not cover command output. If th eoutput (stdout or stderr) contains
+sensitve information, make sure to handle it yourself via C<stdout> and/or C<stderr>
```

Comment:
Should reference the `log` option here? (i.e. recommend setting it to undef?) Generally I am a bit confused as to how this interacts with the log option so it would be good to see that clarified.

Reply:
i'm not sure how to formulate it. the code here does not report the output, but eg `execute` sends it to the real stdout (and that's also why it's not part of the logs, see #245).